### ISSUE TITLE
`scripts` portion of `package.json` contains unneeded and not-so-useful bits

### ISSUE THREAD

**Issue by sogaiu** [sogaiu/tree-sitter-clojure#36]
Currently, `package.json`'s `scripts` section is:
https://github.com/sogaiu/tree-sitter-clojure/blob/262d6d60f39f0f77b3dd08da8ec895bd5a044416/package.json#L6-L9
IIUC, `npx` is unnecessary in this context.

It may be that it forces use of the tree-sitter cli within `node_modules`, but I don't know if this is desirable.

Also, it's unclear whether this invocation:
```
npx tree-sitter generate && npx node-gyp build
```
is that useful.

Possibly, the following is preferrable:
```
tree-sitter generate && node-gyp build
```

Though in practice I think it may be better to do:
```
tree-sitter generate && node-gyp rebuild
```

---

I've been considering the idea of not using `npm` at some point, but I learned that the tree-sitter cli requires a `package.json` file to be at least partially populated to function correctly.

Also, it appears that there is one project that uses tree-sitter-clojure via `yarn`, so for that's another reason to consider retaining `package.json` (though some of its content may be reduced).

It could be that implementing these changes would make changing `scripts` irrelevant as that portion may just be eliminated.

**Comment by sogaiu** [comment]
I went with removing the `scripts` part in 3f8dff3d5bb3288011c2076e345dd7b1e33807f5 (on the dev branch).

**Comment by sogaiu** [comment]
The changes relevant to this issue can now be seen in bf2b6b5b889ba8b1ea205361e34b6a8f3ff52cac.

**Comment by sogaiu** [comment]
As mentioned above, addressed in bf2b6b5.
